test/view/puzzle/puzzle_screen_test.dart

```diff
@@ -415,11 +415,11 @@ void main() {
         await tester.tap(find.byIcon(Icons.help));
 
         // wait for solution replay animation to finish
-        await tester.pump(const Duration(seconds: 3));
+        await tester.pump(const Duration(seconds: 1));
         await tester.pumpAndSettle();
 
         expect(find.byKey(const Key('h4-blackRook')), findsOneWidget);
-        expect(find.byKey(const Key('h8-whiteQueen')), findsNothing);
+        expect(find.byKey(const Key('h8-whiteQueen')), findsOneWidget);
```

Comment:
We could add another check to show the "jump to next move" button is not disabled.